Engine/fgens.c

```diff
@@ -2563,7 +2563,7 @@ static int32_t gen01raw(FGDATA *ff, FUNC *ftp)
     else ftp->nchanls  = 1;
     ftp->flenfrms = ff->flen / ftp->nchanls;  /* VL fixed 8/10/19: using table nchnls */
     ftp->gen01args.sample_rate = (MYFLT) p->sr;
-    ftp->cvtbas = LOFACT * p->sr * csound->onedsr;
+    ftp->cvtbas =  p->sr * csound->onedsr; /* VL - LOFACT factor (1024) removed */
```

Comment:
These are the types of comments I think can be removed. It's noisy and it's only going to confuse future developers IMO.

Reply:
will remove